src/program.rs

```diff
@@ -163,6 +163,32 @@ impl<EC: EvalCache> Program<EC> {
         report(self.vm.import_resolver_mut(), error, self.color_opt)
     }
 
+    /// Wrapper for [`report`].
+    pub fn report_as_str<E>(&mut self, error: E) -> String
+    where
+        E: ToDiagnostic<FileId>,
+    {
+        let contracts_id = self.vm.import_resolver().id_of("<stdlib/contract.ncl>");
```

Comment:
`self.vm.import_resolver()` is used 3 times inside the function. Might it be a tad more readable to store it in a variable?

Reply:
Fixed in 20145575